### ISSUE TITLE
parent sometimes ignores padding for resizing

### ISSUE THREAD

**Issue by greenpixels** [greenpixels/game-maker-ui-library#2]
![grafik](https://user-images.githubusercontent.com/58704291/209013115-a6b42777-3fa4-43a7-8f31-479a6a909cd6.png)

padding 15px
layouting of children is FILL_WIDTH

**Comment by greenpixels** [comment]
This seems to only affect the last element

**Comment by greenpixels** [comment]
The problem could be in layout_children()
The _current_row variable gets increemented when the flex_direction is vertical without ever initializing the first row.
This means that the first entry in the row-array is empty.

```gml
for(var _i = 0; _i < ds_list_size(children); _i++) {
		var _child = ds_list_find_value(children, _i);
		_child.update();
		var _child_w = _child.margin_left + _child.width + _child.margin_right;
		var _child_h = _child.margin_top + _child.height + _child.margin_bottom;
	
		if(check_max_width_exceeded(_child_w) || flex_direction == UI_FLEX_DIRECTION.VERTICAL) {
			// Layout children vertically
			_x = padding_left;
			_y += _current_row_heighest;
			height += _current_row_heighest;
			_current_row_heighest = 0;
			_current_row_width = _child_w;

			_current_row++;  // THIS COULD BE THE ISSUE

		} else {
			// Layout children horizontally
			_current_row_width += _child_w;
		}
	
		_child.x = _x + _child.margin_left;
		_child.y = _y + _child.margin_top;
		_x += _child_w;

		.
		.
		.
```

**Comment by greenpixels** [comment]
The issue seems to have stemmed from "update" being called again after justification of the children, overriding the horizontal position change. Should be fixed in 41e475ac54f6621d182491a5986c06d35d25025b
